cmd/slinky/main.go

```diff
@@ -185,9 +211,36 @@ func runOracle() error {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	cfg, err := config.ReadOracleConfigFromFile(oracleCfgPath)
-	if err != nil {
-		return fmt.Errorf("failed to read oracle config file: %w", err)
+	// Set up logging.
+	logCfg := log.NewDefaultConfig()
+	logCfg.StdOutLogLevel = logLevel
+	logCfg.FileOutLogLevel = fileLogLevel
+	logCfg.DisableRotating = disableRotatingLogs
+	logCfg.WriteTo = writeLogsTo
+	logCfg.MaxSize = maxLogSize
+	logCfg.MaxBackups = maxBackups
+	logCfg.MaxAge = maxAge
+	logCfg.Compress = !disableCompressLogs
+
+	// Build logger.
+	logger := log.NewLogger(logCfg)
+	defer logger.Sync()
+
+	var cfg config.OracleConfig
+	var err error
+	if legacyOracleCfgPath != "" && !useDefaults {
+		logger.Info("The --oracle-config-path flag is deprecated and will be removed in a future release. Please use --default-config --oracle-config instead.")
```

Comment:
I don't really like having `--default-config` be permanent. I think I'd rather just have
`slinky --oracle-config-path ./foobar.json` parse the existing config
`slinky --oracle-config ./foobar.json` parse the new config
`slinky` attempts to parse ./oracle.json as the old config and if not present launches with the defaults.

What do you think? That would make it cleaner when we just remove the old config in a breaking release.